api/v1alpha1/baseboardmanagement_types.go

```diff
@@ -31,8 +31,12 @@ type BootDevice string
 type BaseboardManagementConditionType string
 
 const (
-	On  PowerState = "on"
-	Off PowerState = "off"
+	On     PowerState = "on"
+	Off    PowerState = "off"
+	Status PowerState = "status"
+	Cycle  PowerState = "cycle"
+	Reset  PowerState = "reset"
+	Soft   PowerState = "soft"
```

Comment:
How do "status" and "soft" model a state?

Reply:
True. I was a bit iffy on this. I can rework them to `PowerAction` type which would make more sense.